src/matrix_validator/validator_polars.py

```diff
@@ -176,6 +182,12 @@ def validate_kg_nodes(nodes, limit):
             validation_reports.append(f"Missing required column: {repr(ex)}")
             return validation_reports
 
+        logger.warning(
```

Comment:
Does this get printed even if superfluous_columns is empty? (I couldn't tell if there needs to be superfluous_columns to arrive here or not)

Reply:
I added condition to check if empty.